posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,600 @@
+Jakarta EE 10 Core Profile, Web Profile and Platform are now officially supported in Open Liberty! We'd like start by takin the time to thank all those that provided feedback throughout our various betas.
+
+Jakarta EE 10 marks a major milestone as it is the first release to provide updates to the specifications since Java EE 8 in 2017, and therefore, the first release to provide spec updates since being taken over by the Eclipse Foundation. Among the many updates to existing specifications, it also introduces the Core Profile. The Core Profile is specifically aimed at lightweight runtimes, like Open Liberty, that are optimized for running cloud-native Java microservices.
+ 
+Liberty provides convenience features for running all of the component specifications that are contained in the Jakarta EE 10 Web Profile (`link:{url-prefix}/docs/latest/reference/feature/webProfile-10.0.html[webProfile-10.0]`) and Jakarta EE 10 Platform (`link:{url-prefix}/docs/latest/reference/feature/jakartaee-10.0.html[jakartaee-10.0]`).  For Jakarta EE 10 features in the application client, use the `jakartaeeClient-10.0` Liberty feature.
+
+Using these convenience features enables developers to rapidly develop applications using all of the APIs contained in the Web Profile and Platform specifications.  Unlike the `webProfile-9.1` Liberty feature, the `webProfile-10.0` Liberty feature does not enable the XML Binding feature, because XML Binding is not a Web Profile component specification. If your application relies on XML Binding APIs, you must explicitly enable the `xmlBinding-4.0` Liberty feature when using the `webProfile-10.0` feature.
+
+
+The full list of specifications that make up the three profiles is as follows:
+
+[#coreprofile]
+== Jakarta EE Core Profile 10
+[cols="4,2,3",options="header"]
+|===
+|Specification |Updates |Feature Documentation
+
+[[jsonb]]
+|https://github.com/eclipse/microprofile-open-api/releases/tag/3.1[JSON Binding 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/jsonb-3.0.html[jsonb-3.0]
+
+[[rest]]
+|link:https://jakarta.ee/specifications/restful-ws/3.1/jakarta-restful-ws-spec-3.1.html[RESTful Web Services 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/restfulWS-3.1.html[restfulWS-3.1], link:{url-prefix}/docs/latest/reference/feature/restfulWS-3.1.html[restfulWSClient-3.1]
+
+[[jsonp]]
+|https://github.com/eclipse/microprofile-metrics/releases/tag/5.0[JSON Processing 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/jsonp-2.1.html[jsonp-2.1]
+
+[[cdi]]
+|https://jakarta.ee/specifications/annotations/1.3/[Annotations 1.3] & https://jakarta.ee/specifications/interceptors/2.1/[Interceptors 2.1]
+|Minor updates
+|link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0]
+
+[[cdi]]
+|https://jakarta.ee/specifications/annotations/1.3/[Dependency Injection 2.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0]

```

Comment:
You've mentioned CDI multiple times in this table but not listed it as a core profile spec? The core profile contains 7 specs but there are only 5 rows in the table?
```
    Jakarta Annotations 2.1
    Jakarta Contexts and Dependency Injection (CDI) 4.0 (CDI Lite section)
    Jakarta Dependency Injection 2.0
    Jakarta Interceptors 2.1
    Jakarta JSON Processing 2.1
    Jakarta JSON Binding 3.0
    Jakarta RESTful Web Services 3.1
```

Reply:
CDI Lite is a bit of an oddity.  I tried to handle it via the NOTE under the table, but perhaps that was the wrong way to go and instead I'll move it back into the table and link directly to the CDI Lite portion of the CDI spec.